config/image.go

```diff
@@ -1,6 +1,6 @@
 package config
 
 type Image struct {
-	Command []string `json:"command"`
-	Args    []string `json:"args,omitempty"`
+	Entrypoint []string `json:"entrypoint,omitempty"`
+	Cmd        []string `json:"cmd,omitempty"`
```

Comment:
Would it be better to stick with Cmd?

Reply:
The behaviour has changed, so I wanted to change it.